src/mixer/basetrackplayer.cpp

```diff
@@ -548,6 +557,13 @@ void BaseTrackPlayerImpl::slotCloneFromDeck(double d) {
     }
 }
 
+void BaseTrackPlayerImpl::slotCloneFromSampler(double d) {
+    int sampler = std::lround(d);
+    if (sampler >= 1) {
+        slotCloneFromGroup(PlayerManager::groupForSampler(sampler-1));
```

Comment:
```suggestion
        slotCloneFromGroup(PlayerManager::groupForSampler(sampler - 1));
```

Reply:
I know but in this case I chose to keep it consistent with existing `CloneFromDeck`